### PR TITLE
Feature intro

### PR DESCRIPTION
**Pull request by sofiatengvall**

### CODE REVIEW

File: client/src/actions.js [hunk 1]
```diff
@@ -17,6 +19,10 @@ export function loadComments(comments) {
   return createAction(LOAD_ALL, { comments })
 }
 
+export function introDone() {
```
Comment (HenrikHelenius) [inline client/src/actions.js]: rename ...Completed

File: client/src/components/onboarding/onboarding.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+          `
+        )
+      )
+    } else if (step === 3) {
+      return (
+        makeContent(
+          'Survey Panel',
+          dev
+            ? `
+            This is the survey panel where the users can aswer your questions.
```
Comment (HenrikHelenius) [inline client/src/components/onboarding/onboarding.js]: typo

File: client/src/components/onboarding/onboarding.js [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+            `
+            : `
+            This is the survey panel where you can aswer questions.
```
Comment (HenrikHelenius) [inline client/src/components/onboarding/onboarding.js]: typo

File: client/src/components/onboarding/onboarding.js (hunk 2 above)
Comment (HenrikHelenius) [reply client/src/components/onboarding/onboarding.js]: also, you can create surveys?

File: client/src/components/onboarding/onboarding.scss [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  flex: 0;
+  padding: 0 $padding;
+
+  button {
+    @extend %icon-button;
+    margin: 0;
+  }
+
+  .next-button {
+    transform: scaleX(-1);
```
Comment (HenrikHelenius) [inline client/src/components/onboarding/onboarding.scss]: ööh what

**Review by HenrikHelenius — DISMISSED** [review]

File: client/src/components/onboarding/onboarding.scss (hunk 4 above)
Comment (sofiatengvall) [reply client/src/components/onboarding/onboarding.scss]: kääntää et on nuoli toiseen suuntaan

File: client/src/components/onboarding/onboarding.js (hunk 2 above)
Comment (sofiatengvall) [reply client/src/components/onboarding/onboarding.js]: do we create them in the panel?

File: client/src/components/open-comment-panel-button/open-comment-panel-button.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+// Helpers
+import classNames from 'classnames/bind'
+
+// Styles
+import styles from './open-comment-panel-button.scss'
+
+const css = classNames.bind(styles)
+
+const OpenCommentPanelButton = ({ hidden, onClick }) => (
```
Comment (tobbe-j) [inline client/src/components/open-comment-panel-button/open-comment-panel-button.js:10]: This should be merged with survey-panle-button and changed to a standard open-button component as they are the same except for the text

File: client/src/index.js [hunk 6]
```diff
@@ -79,13 +89,16 @@ class MainView extends React.Component {
...
+    this.handleCommentPanelClick = this.handleCommentPanelClick.bind(this)
     this.toggleTagElementState = this.toggleTagElementState.bind(this)
     this.handleElementTagged = this.handleElementTagged.bind(this)
     this.unsetTaggedElement = this.unsetTaggedElement.bind(this)
 
     this.state = {
       surveyPanelIsHidden: true,
       surveyButtonIsHidden: false,
+      commentPanelIsHidden: true,
+      commentPanelButtonIsHidden: false,
```
Comment (tobbe-j) [inline client/src/index.js]: Could these be only one state? as they always change together

**Review by tobbe-j — APPROVED** [review]
👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

**Pull request merged**